checks/compound/check-compound-proposal-details.ts

```diff
@@ -144,29 +145,27 @@ async function getTransactionMessages(chain: CometChains, proposalId: number, tr
       console.error(
         `No functions found for ContractName - ${contractNameAndAbi.contractName}. FunctionSignature - ${functionSignature}. TransactionFormatter - ${transactionFormatter}. FunctionMapping - ${functionMapping}`
       )
-      return { info: `**${target}.${functionSignature} called with :** (${decodedCalldata.join(',')})` }
+      const aiSummary = await generateAISummary(chain, target, functionSignature, decodedCalldata)
+      return { info: aiSummary }
     }
 
-    if (!transactionFormatter) {
-      return { info: `**${target} - ${functionSignature} called with :** (${decodedCalldata.join(',')})` }
-    } else {
-      const [contractName, formatterName] = transactionFormatter.split('.')
-      console.log(`GetFormatter: ContractName - ${contractName} and FormatterName - ${formatterName}`)
-      const formattersLookupElement = formattersLookup[contractName]?.[formatterName]
-
-      const message = !formattersLookupElement
-        ? `${target}.${functionSignature} - called with ${decodedCalldata}`
-        : await formattersLookupElement(
-            chain,
-            transactionInfo,
-            decodedCalldata.map((data) => data.toString())
-          )
-      return { info: message }
-    }
+    const [contractName, formatterName] = transactionFormatter.split('.')
+    console.log(`GetFormatter: ContractName - ${contractName} and FormatterName - ${formatterName}`)
+    const formattersLookupElement = formattersLookup[contractName]?.[formatterName]
+    
+    const message = !formattersLookupElement
+      ? `${target}.${functionSignature} - called with ${decodedCalldata}`
```

Comment:
Use if statement instead of ternary operator. 

We don't need to have common `message` const.

from if-else, we can return `{ info: message }`